src/App.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Todo from './pages/Todo';
+
+
+
```

Comment:
a lot of empty lines

Reply:
Do we really need `App.tsx`?  Maybe better render `Todo` component in index.tsx?